azure-pipelines/auth-client/common-instrumented-tests.yml

```diff
@@ -0,0 +1,10 @@
+# File: azure-pipelines\auth-client\common-instrumented-test.yml
```

Comment:
Should we create these files in android-complete, to start consolidating our pipelines/yml files in one place.?

Reply:
I was afraid of this comment. jeje, yeah let me rework it.

Reply:
Tested here 
https://github.com/AzureAD/android-complete/pull/190
https://identitydivision.visualstudio.com/Engineering/_build?definitionId=1863

Reply:
synced offline